ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,208 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  ashenEyeDirections?: string[];
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheThirdCircle,
+  triggers: [
+    {
+      id: 'P3N Experimental Fireplume 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'P3N Experimental Fireplume 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.outOfMiddle!();
+      },
+      outputStrings: {
+        outOfMiddle: {
+          en: 'Out Of Middle',
+          de: 'Raus aus der Mitte',
+          ja: '横へ',
+          cn: '远离中间',
+          ko: '가운데 피하기',
+        },
+      },
+    },
+    {
+      id: 'P3N Scorched Exaltation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B8', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Heat of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B2', source: 'Phoinix' }),
+      suppressSeconds: 1,
+      response: Responses.tankBuster(),
```

Comment:
```suggestion
      response: Responses.tankCleave(),
```

At least one of my logs has an instance of this being a splash.

Reply:
Maybe `tankCleave('alert')`? Maybe it should be alert by default to match buster, but it's not right now.

Reply:
Ah, that’s a good point, yes. I think you’re right that it should be `alert` by default.

Reply:
Fixed in e32226eea0497fd9f12913a9d7136930d24c0a28